python/cudf/udf_cpp/strings/src/strings/udf/udf_apis.cu

```diff
@@ -101,6 +101,8 @@ void free_udf_string_array(cudf::strings::udf::udf_string* d_strings,
 
 // external APIs
 
+int get_cuda_build_version() { return CUDA_VERSION; }
```

Comment:
This can probably be just a static int rather than a function. But if we use a function, it would be possible to do the version parsing in C++ and just return a pair of (major, minor) to Cython. Which would be better?

Reply:
Since it's lightweight either way I prefer to parse in python where it seems easier to break and debug if necessary.

Reply:
Sounds good! Resolving.